handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -313,31 +333,58 @@ public void operationComplete(Future<SslContext> future) throws Exception {
     }
 
     /**
-     * The default implementation will simply call {@link AsyncMapping#map(Object, Promise)} but
-     * users can override this method to implement custom behavior.
+     * The default implementation will simply call {@link SslContextMapping#map(String, Promise)} but users can
```

Comment:
Either Intellij is confused or javadoc references are subject to type erasure. Based upon a reference in the JDKs [HashMap](http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/util/HashMap.java#l552) I suspect its the latter. Can you confirm this?